components/Dropdown/Dropdown.less

```diff
@@ -24,6 +24,7 @@
     border: 2.5px solid @lambda-red;
   }
   .dropdown-link {
+    transition: opacity 2s ease-in;
     width: 100%;
     text-align: center;
```

Comment:
If you want the dropdown menu to have a fade-in animation, you could transfer line 27 to your `.dropdown-content` selector instead (apply the transition to the container itself). Also, consider shortening the duration of this transition.